Core/GDCore/Project/EventsFunction.h

```diff
@@ -116,6 +117,24 @@ class GD_CORE_API EventsFunction {
    */
   FunctionType GetFunctionType() const { return functionType; };
 
+  /**
+   * \brief Returns true if the function is private.
+   */
+  bool IsPrivate() {
+    if (isPrivate)
```

Comment:
I probably did a c++ newbie mistake again, but this is the only way I got this to work. `return isPrivate;` always returned true no matter what for some reason.

Reply:
Weird, it's very unlikely for the compiler to make a miscompilation of something that simple. Can you try going back to `return isPrivate;`?

Reply:
My computer must have had some sort of problem, it seems I had a lot of very weird issues that just disappeared. This works now.